### DISCUSSION TITLE
Site Authentication

### DISCUSSION THREAD

**Discussion by Domeman** [wintercms/winter#291] [Q&A] [answered]
Is there a front-end authentication package for wither similar to laravel?

**Comment by LukeTowers** [comment, marked as answer]
Yes, https://github.com/wintercms/wn-user-plugin

**Reply by Domeman** [reply]
Install wn-user-plugin. it just creates users on the backend. 
I can not create a user because it asks for the password but there is no password field.
There is no login on the front end.

**Reply by bennothommo** [reply]
You need to read the README (https://github.com/wintercms/wn-user-plugin/blob/main/README.md) as it will tell you how to implement the login into your CMS templates.

**Reply by Domeman** [reply]
I would have to implement the login page, forgot password, etc. Unlike laravel where they generate the pages.

**Reply by LukeTowers** [reply]
See https://github.com/rainlab/vanilla-theme/tree/master/pages for how to do that, it's super easy since the functionality is all contained in the `account` component.
